swift/StableDiffusionCLI/main.swift

```diff
@@ -193,22 +195,19 @@ struct StableDiffusionSample: ParsableCommand {
         pipelineConfig.guidanceScale = guidanceScale
         pipelineConfig.schedulerType = scheduler.stableDiffusionScheduler
         pipelineConfig.rngType = rng.stableDiffusionRNG
-
-        if isXL {
-            pipelineConfig.encoderScaleFactor = 0.13025
-            pipelineConfig.decoderScaleFactor = 0.13025
-        }
+        pipelineConfig.useDenoisedIntermediates = true
```

Comment:
It's only applicable if also using the --save-every parameter. Other than that it shouldn't have any effect since the latents will be denoised at the end of the generation anyway.